xml/System.Text.Json/JsonElement.xml

```diff
@@ -741,10 +752,24 @@ For other types, the value of <xref:System.Text.Json.JsonElement.GetRawText> is
         <Parameter Name="value" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">When the method returns, contains the byte representation of <paramref name="value" /> if the method succeeded.</param>
```

Comment:
Also, this comment doesn't make sense. value doesn't contain the byte[] representation of value, it contains the byte[] representation of the json element.

```suggestion
        <param name="value">When the method returns, contains the byte[] representation of the current JSON string if the method succeeded.</param>
```

Reply:
Also, we may want the Try and the non-Try method to have similar looking documentation.

Reply:
cc @bartonjs

Reply:
I used "byte array representation"

Reply:
That's fine, but you missed the other feedback which was change `<paramref name="value" />` to `the current JSON string`